tools/packaging/common/istio-iptables.sh

```diff
@@ -264,12 +265,20 @@ if [ -z "${PROXY_GID}" ]; then
 PROXY_GID=${PROXY_UID}
 fi
 
-POD_IP=$(hostname --ip-address)
-# Check if pod's ip is ipv4 or ipv6, in case of ipv6 set variable
-# to program ip6tables
-if isIPv6 "$POD_IP"; then
-  ENABLE_INBOUND_IPV6=$POD_IP
-fi
+# Check if any of the pod's ip addresses are ipv6. If so, set variable
```

Comment:
Wow I am surprised we are still using istio-iptables.sh here. Does that mean CNI has completely drifted from every iptables change made to Istio recently? Is there any efforts to use the golang implementation, and is this change needed there?

Reply:
I know that we use the go iptables implementation in the validation code, and we use it in the distroless image (when we don't have a bash to execute). 

I'm not sure if this change is needed in the go implementation.

I think we need to move cni into istio/istio, so we could work on migrating to the go binary when we make that change. That would support the effort to really get cni into wider use.

Reply:
(to clarify -- the distroless image with go binary passes some tests, but I don't know that they are comprehensive, and I'm not sure if anyone actually uses those in production)

Reply:
Actually, ignore that -- I'm confusing the install-cni container with the sidecar. I don't think istio-cni has ever been converted to use the go binary, but the sidecar image does use it for validating that the cni iptables rules have been applied correctly.

Reply:
@howardjohn this has not drifted yet AFAIK. But that's definitely a high risk.